src/index.js

```diff
@@ -0,0 +1,161 @@
+import {
+  ADVANCE_STANDARD,
+  LENGTH,
+  MAX_NUMBER,
+  MIN_NUMBER,
+  MIN_RACING_COUNT,
+} from "./constants.js";
+import { createElement } from "./utils.js";
+
+class Car {
+  constructor(name, location) {
+    this.name = name;
+    this.location = location;
+  }
+
+  advance() {
+    this.location += "-";
+  }
+}
+
+class RacingGame {
+  constructor() {
+    this.cars = [];
+
+    this.init();
+  }
+
+  init() {
+    this.carNamesSubmitBtn = document.getElementById("car-names-submit");
+    this.racingCountSubmitBtn = document.getElementById("racing-count-submit");
+    this.carNamesInputEl = document.getElementById("car-names-input");
+    this.racingCountInputEl = document.getElementById("racing-count-input");
+
+    createElement("app", "div", null, "race-result");
+    this.handleCarNamesSubmit();
+    this.handleRacingCountSubmit();
+  }
+
+  addCar(name) {
+    const INIT_LOCATION = "";
+    const car = new Car(name, INIT_LOCATION);
+    this.cars.push(car);
+  }
```

Comment:
```suggestion
  addCar(name) {
    const INIT_LOCATION = "";
  	this.cars = [...this.cars, new Car(name, INIT_LOCATION)];
  }
```

이렇게 작성할수도 있습니다~ 개인 취향적인 부분이긴 하지만, 전 이게 더 편하더라구요~